### PR TITLE
[POS-341]Fixed typo in comments and removed one of the unit tests

### PR DESCRIPTION
**Pull request by navidmatin**

### CODE REVIEW

**Comment by navidmatin** [comment]
related to https://github.com/TheLevelUp/pos-proposed-orders-java/pull/5

File: LevelUp.Pos.ProposedOrders.Tests/CalculatorTests.cs [hunk 1]
```diff
@@ -60,25 +60,5 @@ private void RunTestArray(int[,] values)
                     i, expectedCheckValues, actualCheckValues, Environment.NewLine);
             }
         }
-
-        [TestMethod]
-        public void TestTest123()
```
Comment (devlaf) [inline LevelUp.Pos.ProposedOrders.Tests/CalculatorTests.cs]: Did this test ever have any value?  Does it test anything not covered by the "test battery" stuff?

Comment (navidmatin) [inline LevelUp.Pos.ProposedOrders.Tests/CalculatorTests.cs]: Not really, it is just one case when TestBattery stuff goes through multiple cases.

**Comment by devlaf** [comment]
ok, LGTM!

**Pull request merged**